util/chplenv/utils.py

```diff
@@ -1,8 +1,24 @@
-""" Utility functions for chplenv modules """
+"""Utility functions for chplenv modules"""
+import os
 import subprocess
+import sys
+
+CHPL_DEVELOPER = os.getenv('CHPL_DEVELOPER')
```

Comment:
I would also prefer that this logic get stuffed into a "get_chpl_developer()` routine instead of being a global var and  in case we add a --developer flag to printchplenv in the future or something